docs/plugins/repository-s3.asciidoc

```diff
@@ -196,6 +196,9 @@ The following settings are supported:
     Specifies the path within bucket to repository data. Defaults to
     value of `repositories.s3.base_path` or to root directory if not set.
 
+    Please note that although the base path can be started with a "/",
+    this has now been deprecated.
```

Comment:
I believe deprecated implies this well - if something has been deprecated it is the official term for something that will be no longer be supported in the future.